Formula/trezor-bridge.rb

```diff
@@ -0,0 +1,34 @@
+class TrezorBridge < Formula
+  desc "Trezor Communication Daemon"
+  homepage "https://github.com/trezor/trezord-go"
+  url "https://github.com/trezor/trezord-go/archive/refs/tags/v2.0.31.tar.gz"
+  sha256 "fd834a5bf04417cc50ed4a418d40de4c257cbc86edca01b07aa01a9cf818e60e"
+  license "LGPL-3.0-only"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", *std_go_args(output: bin/"trezord-go", ldflags: "-s -w")
+  end
+
+  plist_options startup: true
+
+  service do
+    run opt_bin/"trezord-go"
+    keep_alive true
+    working_dir HOMEBREW_PREFIX
+  end
+
+  test do
+    # start the server with the USB disabled
+    server = IO.popen("#{bin}/trezord-go", "-u=false")
```

Comment:
Can we pick a port for the server?

Reply:
Yes we can. The default is 21325, should I change it to something else?

Reply:
It would be much preferable since otherwise it could conflict.

Reply:
I created [a PR to the upstream project](https://github.com/trezor/trezord-go/pull/280), but I don't want to block merge of this PR waiting for the next upstream release.